examples/run_aims.py

```diff
@@ -60,6 +60,6 @@ def test_run_aims():
     from unittest import TestCase
     with TestCase.assertRaises(KeyError):
```

Comment:
Hi Andy, I think this `assertRaises` may have to be inside a function of a subclass of `unittest.TestCase`, as its first argument is `self`
https://docs.python.org/3/library/unittest.html

Reply:
This should not be the case, it is just a function in the TestCase class that can be imported.

Reply:
I've thought further. We can't mix unittest and pytest (which we use, https://docs.pytest.org/en/stable/), it'll be very confusing for future users. 

I've dug down and found examples with pytest for checking Exceptions here: https://pytest-with-eric.com/introduction/pytest-assert-exception/#Using-Pytest-Asserts-To-Assert-Exceptions, which should be the basis of the test we use.